stdlib/2/sets.pyi

```diff
@@ -0,0 +1,60 @@
+# Stubs for sets (Python 2)
+from typing import Any, Callable, Hashable, Iterable, Iterator, Mapping, Optional, TypeVar, Union
+
+_T = TypeVar('_T')
+Setlike = Union['BaseSet', Iterable]
+
+class BaseSet(Iterable[_T]):
+    def __init__(self) -> None: ...
+    def __len__(self) -> int: ...
+    def __repr__(self) -> str: ...
+    __str__ = ...  # type: Callable[[BaseSet[_T]], str]
+    def __iter__(self) -> Iterator[_T]: ...
+    def __cmp__(self, other: Any) -> None: ...
+    def __eq__(self, other: Any) -> bool: ...
+    def __ne__(self, other: Any) -> bool: ...
+    def copy(self) -> BaseSet[_T]: ...
```

Comment:
I think this should be `copy(self: _SelfT) -> _SelfT` (where `_SelfT` is a new typevar, with a bound on `BaseSet[_T]`): it returns an instance of the same class.